scripts/preferences_cleaner/preferences_cleaner.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```

Comment:
This is not yet supported by th etest suite, you should extend this mechanism: https://github.com/cyberbotics/webots/blob/c4ce84200caa0725655d1538076099d2fc036a22/tests/sources/test_license.py#L70

Reply:
Mmmm, I finally changed this to `python`. The version is tested later and described in the README.md. It sounds better to me.